node/src/components/consensus/highway_core/state.rs

```diff
@@ -496,39 +485,11 @@ impl<C: Context> State<C> {
             return;
         }
```

Comment:
This is also obsolete now, since it's implicit in `merge`.
(It's also wrong, I think: If the creator is faulty, the left side will be `0`, but even if the right one is `0`, too, we still should merge the panoramas.)

Reply:
How can creator be faulty and get his unit endorsed by majority of validators?

Reply:
Ah, if the unit comes before the fork/equivocation.